stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -141,6 +157,7 @@ def landing_page(self, **kwargs) -> LandingPage:
             id=self.landing_page_id,
             title=self.title,
             description=self.description,
+            conformsTo=self.list_conformance_classes(),
```

Comment:
This looks like it will fix https://github.com/stac-utils/stac-fastapi/issues/169

Reply:
@rsmith013  I thought this would fix #169, but when I ran this using docker-compose from your branch, the landing page `conformsTo` is still empty. maybe i'm not running it right? This is a pretty major bug in stac-fastapi, so I just wanted to make sure it gets fixed. Do you see `conformsTo` in the landing page when you run it locally?

Reply:
yes:
![image](https://user-images.githubusercontent.com/22395703/124885448-2c542800-dfcb-11eb-87d7-9f463f056c1d.png)

Reply:
@philvarner You may need to rebuild the images when running `docker-compose` to see these changes:

```console
$ docker-compose up --build ...
```